lib/livebook/hubs/fly.ex

```diff
@@ -162,4 +162,10 @@ defimpl Livebook.Hubs.Provider, for: Livebook.Hubs.Fly do
   def delete_secret(_fly, _secret), do: :ok
 
   def connection_error(_fly), do: raise("not implemented")
+
+  def notebook_stamp(_hub, _notebook_source, _metadata) do
+    :skip
```

Comment:
```suggestion
    raise "not implemented"
```

Reply:
no, it will just crash. we will fix it later.

Reply:
Wait what, it would crash when saving notebook with any of these hubs, no?

Reply:
It will, and then we will fix it. Unless it will break tests, then `:skip` is fine.

Reply:
We test that we can store hub id, so we need a test with other hub yeah :D